packages/server/test/support/fixtures/projects/e2e/cypress/integration/issue_1987_spec.js

```diff
@@ -0,0 +1,13 @@
+describe('suite', () => {
+  before(() => {
+    // will cause infinite top navigation
+    cy.visit('http://localhost:3434')
```

Comment:
I don't understand how this test could ever work. This will be considered visiting domains of 2 different origins in a single test. What am I missing?

<img width="530" alt="Screen Shot 2020-04-17 at 3 01 25 PM" src="https://user-images.githubusercontent.com/1271364/79549061-71392980-80bc-11ea-8879-fabea4289abd.png">

Reply:
look at the order of the tests. The before hook is tied to an empty test, so the next test only has one navigation

Reply:
tricky 👍